ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,14 @@
+[[!meta title="Rename an account"]]
+
+We should sparingly rename an user account, and only do so when it is
+necessary (e.g. typo in the username).
+
+To rename a individual account, `sorry` the account with the old name and
+make a `note` of why you are changing the username. Then manually create an
```

Comment:
I think it would be good to be explicit about how to do this (IOW, please give an example command line).